modules/moving.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-unresolved
+import Task from './task.js';
+
+const task = new Task();
+// eslint-disable-next-line no-undef
```

Comment:
- [Optional] It is advisable not to disable linters but rather to find a way to solve the issue. Linters force you to write code using best practices, this will make you a great developer.